banzai/tests/test_mosaic_maker.py

```diff
@@ -9,6 +9,11 @@ class FakeMosaicImage(FakeImage):
     def __init__(self, *args, **kwargs):
         super(FakeMosaicImage, self).__init__(*args, **kwargs)
         self.extension_headers = None
```

Comment:
Still necessary to have `self.extension_headers` initialized to None here?

Reply:
Fixed.